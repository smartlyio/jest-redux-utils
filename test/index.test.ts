import reducer, {setA, setB } from "./reducer";
import reducerTest from "../src/index";

describe("reducer", reducerTest(reducer)(function () {
  this.snapshotAction(setA("other a"));
  this.snapshotActionSequence(setB({ c: true }), setA("some other"));
}));