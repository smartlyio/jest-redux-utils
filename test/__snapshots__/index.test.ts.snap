// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reducer SET_A("other a") matches snapshot 1`] = `
Object {
  "a": "other a",
  "d": 0,
}
`;

exports[`reducer matches snapshot 1`] = `
Array [
  Object {
    "a": "a",
    "b": Object {
      "c": true,
    },
    "d": 0,
  },
  Object {
    "a": "some other",
    "b": Object {
      "c": true,
    },
    "d": 0,
  },
]
`;
